CarRental/CarRental.Web/Views/Home/_AvailableOptionsForYearFrom.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@ViewData.Eval("Year")
+@*@model dynamic*@
+
+@{
+    List<int> years = new();
+    int presentYear = DateTime.Now.Year;
+
+    for (int year = presentYear - 15; year <= presentYear; year++)
+    {
+        years.Add(year);
+    }
+
+    string modelAlert = @Model == null ? "Model=null" : "Model != null";
+
+}
+<option value="">-Production year-</option>
+
+@foreach (var year in years)
+{
+    if (Model != null)
+    {
+        bool select = year == Model.SearchDto.ProductionYearFrom;
+        <option value="@year" selected=@select>@year</option>
+    }
+    else
+    {
+        <option value="@year">@year</option>
+    }
+}
```

Comment:
brak pustej linii :)
(a skoro już mamy .editconfig w solucji to być może tego typu rzeczy rozwiąże VS menu -> Analyze -> Code Cleanup -> Run Code Cleanup)